content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,33 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+When scaling down, the ReplicaSet controller chooses which pods to delete by sorting the available pods to
+prioritize scaling down pods based on the following sorting keys:
+ 1. If only one of the pods is assigned to a node, the pod that is not
+    assigned comes before the pod that is.
+ 2. If the pods' phases differ, a pending pod comes before a pod whose phase
+    is unknown, and a pod whose phase is unknown comes before a running pod.
+ 3. If exactly one of the pods is ready, the pod that is not ready comes
+    before the ready pod.
+ 4. If controller.kubernetes.io/pod-deletion-cost annotation is set, then
```

Comment:
FYI @ahg-g 

Maybe we can have a single docs PR for both KEP

Reply:
sounds good.

Reply:
actually lets keep them separate, the other PR is ready for review.

Reply:
BTW I'd be happy to see this level of detail covered in the reference section of the docs.